docs/application/dotnet/guides/app-management/application-preference.md

```diff
@@ -0,0 +1,266 @@
+# Application Preferences
+
+You can manage application preferences by setting and getting them. You can also share stored preference data among applications in the same package.
+
+## Managing Application Preferences
+
+To manage preferences:
+
+- To store and retrieve variables:
```

Comment:
Can this section be broken down into sub-headings for readability, the unordered list seems to be tough to follow.

### Store and retrieve 
....
###  List records
...
###  Remove records